### PR TITLE
mpd: update to 0.22.6.

### PR DESCRIPTION
**Pull request by UsernameRandomlyGenerated**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/mpd/template [hunk 1]
```diff
@@ -37,7 +37,7 @@ license="GPL-2.0-or-later"
 homepage="https://www.musicpd.org/"
 changelog="https://raw.githubusercontent.com/MusicPlayerDaemon/MPD/v${version}/NEWS"
```
Comment (ericonr) [inline srcpkgs/mpd/template:38]:
```suggestion
changelog="https://raw.githubusercontent.com/MusicPlayerDaemon/MPD/master/NEWS"
```

**Review by ericonr — COMMENTED** [review]
Have you looked at the test failure for musl? It was present on 0.22.4 already, so no reason to hold the update on it, but would be nice to get fixed one way or another.

File: srcpkgs/mpd/template (hunk 1 above)
Comment (tibequadorian) [reply srcpkgs/mpd/template:38]: I think it's because musl strptime does not support the `%F` field descriptor

**Comment by tibequadorian** [comment]
> Have you looked at the test failure for musl? It was present on 0.22.4 already, so no reason to hold the update on it, but would be nice to get fixed one way or another.

I think it's because musl strptime does not support the `%F` field descriptor

**Comment by ericonr** [comment]
Nice find! I will try to write up a patch.

**Comment by ericonr** [comment]
Pull in https://github.com/MusicPlayerDaemon/MPD/pull/1110

**Pull request closed without merging**